bftengine/src/bftengine/ControlStateManager.cpp

```diff
@@ -43,7 +43,11 @@ std::optional<int64_t> ControlStateManager::getCheckpointToStopAt() {
   std::istringstream inStream;
   inStream.str(scratchPage_);
   controlStateMessages::StopAtNextCheckpointMessage msg;
-  concord::serialize::Serializable::deserialize(inStream, msg);
+  try {
+    concord::serialize::Serializable::deserialize(inStream, msg);
+  } catch (std::exception& e) {
```

Comment:
We should definitely at least log log an error here. But it may make even more sense to abort the process or just let the exception be thrown and not caught. 

Is there any normal case reason that deserialization will fail? If there is it's a bug, as we need to know that in this case the storage is corrupt.

Reply:
If the reserved page is empty, the serializer throws an error.
This may happen if we try to read the checkpoint before we set anything into there (or if we cleared the reserved page)

Reply:
I will add a log message though

Reply:
done